### PR TITLE
Some optimizations

### PR DESCRIPTION
**Pull request by adrientetar**
cc #79 

r? @anthrotype

### CODE REVIEW

File: Lib/defcon/objects/glyph.py [hunk 1]
```diff
@@ -161,6 +163,11 @@ def __del__(self):
         self._lib = None
         self._image = None
 
+    def postNotification(self, notification, data=None):
+        if self._isLoading:
+            return
+        super().postNotification(notification, data)
```
Comment (anthrotype) [inline Lib/defcon/objects/glyph.py]: that's python3-only, you need `super(ClassName, self).methodName(*args, **kwargs)`

**Comment by anthrotype** [comment]
you sure you gain so much by moving the import at the top? isn't it that the subsequent time you import in a scope the module is simply looked up?
anyway, lgtm

**Comment by adrientetar** [comment]
> you sure you gain so much by moving the import at the top? isn't it that the subsequent time you import in a scope the module is simply looked up?

getPointPen is called a lot so it can matter more when loading a large font. Anyway this is trivial stuff I saw while profiling, I made a list of more in-depth changes.

**Comment by adrientetar** [comment]
Actually a larger-scale solution can be provided here I think.

**Pull request closed without merging**